build.gradle.kts

```diff
@@ -310,7 +310,7 @@ tasks {
 	javadoc {
 		if (releaseBuild) {
 			javadocTool.set(project.javaToolchains.javadocToolFor {
-				// create Javadoc with least Java version to get all features
```

Comment:
"latest"?

Reply:
Actually I think this means that we use the smallest Java version we need for all the features we want to use but it's phrased a bit weird?

Reply:
Then maybe something like:

```java
// create Javadoc with the minimum Java version needed for our desired features (e.g. search)
```